connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -387,4 +411,12 @@ boolean isHeartbeatTopic(String topic) {
     String formatRemoteTopic(String topic) {
         return replicationPolicy.formatRemoteTopic(sourceAndTarget.source(), topic);
     }
+
+    void setKnownTopicPartitions(List<TopicPartition> knownTopicPartitions) {
```

Comment:
I see these are only for the unit tests. Would it be cleaner to add a new constructor?

Reply:
a new constructor works fine to me 👍